.circleci/config.yml

```diff
@@ -21,11 +21,21 @@ jobs:
       - run:
           name: Install Dependencies
           command: yarn
+      - run:
+          name: Create Linux CLI binary
+          command: HOST=host yarn build.binary
```

Comment:
what does this host bit mean?

Reply:
just updated the readme with:
```
`HOST` variable is used to specify what binary we want to build.
`HOST` might be set to anything that is considered a target in `pkg` [supports](https://github.com/zeit/pkg#targets).
```

https://github.com/stoplightio/prism/blob/SO-145_add_test_harness/test-harness/README.md#alternatively-run-any-of-the-following

LMK if still not clear :)